src/faq.md

```diff
@@ -238,16 +238,16 @@ long enough for runtime optimizations to be applied.
 
 ### Q. What browsers do you support as JavaScript compilation targets?
 
-The _production_ compiler ([dart2js][]) supports
-the last two major releases of the following browsers:
+The _production_ web compiler supports the last two major releases of
+the following browsers:
 
   * Chrome
   * Edge
   * Firefox
   * Safari
 
-The _development_ compiler ([dartdevc][]) supports only Chrome.
-You _might_ be able to use other modern browsers
+The _development_ web compiler supports Chrome only.
+You _might_ be able to use other modern browsers based on Chromium
```

Comment:
I'd recommend reverting this line change - only Edge is based on chromium, FF and Safari are not.